libvast/include/vast/type.hpp

```diff
@@ -58,7 +61,7 @@ using concrete_types
 
 /// A concept that models any concrete type.
 template <class T>
-concept concrete_type = requires(const T& value) {
+concept concrete_type = requires(const T& value, arrow::MemoryPool* pool) {
```

Comment:
```suggestion
concept concrete_type = requires(const T& value) {
```

A relict from an older version of this concept that was more strict, but sadly introduced a dependency on `vast/data.hpp`.